py3status/modules/online_status.py

```diff
@@ -21,13 +20,7 @@
 @author obb
 """
 
-import os
-import subprocess
-try:
-    # python3
-    from urllib.request import urlopen
-except:
-    from urllib2 import urlopen
+from urllib.parse import urlsplit
```

Comment:
> UnicodeEncodeError: 'ascii' codec can't encode character u'\xb0' in position 0: ordinal not in range(128)

:) this is why we have all the helper functions eg `safe_format()` because that sorts all this crap for us.  but in this example you are trying to format an ascii string with unicode.

I made an issue #699